.circleci/config.yml

```diff
@@ -1,827 +1,56 @@
 version: 2.1
 
 references:
-  - &OTP24 cimg/elixir:1.12  # Contains Erlang 24.0
+  - &CERT_KEY certs-cache-{{ checksum "tools/ssl/Makefile" }}-{{ checksum "cert_date" }}-v5
```

Comment:
Why do we need the checksum of the cert_date? I think it'd make it more readable if it was just the cert_date here, so that we see on the build logs which cache are we trying to fetch.